src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -179,6 +179,51 @@ describe('RuleInformationProvider', () => {
         expect(ruleInformation.getResultStatus(ruleResult)).toBe('fail');
     });
 
+    test('getRuleInformation returns correct data for ClassNameCheck rule', () => {
```

Comment:
Could we have a `it.each` for these test cases (L182-L225) to reduce the redundancy?

Reply:
Done!